tests/gate-tests.lisp

```diff
@@ -112,6 +112,20 @@
       (run qvm)
       (is (double-float= 1 (probability (aref amps 0)) 1/10000)))))
 
+(deftest test-xy-gate ()
+  "Test that XY gates are supported, and are equivalent to PISWAP gates."
+  (dotimes (i 10)
+    (let ((amps (qvm::randomize-wavefunction (qvm::wf 1.0 0.0 0.0 0.0))))
+      (flet ((eval-prog (quil-string)
+               (let ((qvm (make-qvm 2)))
+                 (load-program qvm (with-output-to-quil
+                                     (format t quil-string)))
```

Comment:
no need to `format` here if you remove the `~%` from the `quil-string`s. `with-output-to-quil` will `write-line` for any bare strings you give it.